README.md

```diff
@@ -94,3 +94,30 @@ This enables `buffer-face-mode` with the chosen font when an APL
 buffer is opened.
 
 This may be changed to be the default in a future version.
+
+Customizing keyboard layout
+---------------------------
+Sometimes the default keyboard layout may not be enough, for example for
+Dvorak users. In this case one can customize it by redefining 2 variables:
```

Comment:
The word "one" isn't needed here. Something like "It can be customised by..."

Also, let's stick with International English spelling.